frontend/demo/src/pages/Inbox/Messenger/MessageList/index.tsx

```diff
@@ -54,21 +56,49 @@ const MessageList = (props: MessageListProps) => {
     scrollBottom();
   }, [currentConversationId]);
 
+  useEffect(() => {
+    setCurrentConversation(conversations.find(item => item && item.id === currentConversationId));
+  }, [currentConversationId, conversations]);
+
   const scrollBottom = () => {
     messageListRef.current.scrollTop = messageListRef.current.scrollHeight;
   };
 
+  const isUser = (message: Message) => message.senderType !== SenderType.appUser;
```

Comment:
This seems the wrong way around, or it should be called `isContact` 🤔 Sorry for the confusing data model. We will make it binary!